test/e2e/node/pods.go

```diff
@@ -384,14 +385,22 @@ var _ = SIGDescribe("Pods Extended", func() {
 				framework.Failf("pod spec TerminationGracePeriodSeconds is not 1: %d", *pod.Spec.TerminationGracePeriodSeconds)
 			}
 
-			time.Sleep(5 * time.Second)
-
-			pod, err = podClient.Get(ctx, pod.Name, metav1.GetOptions{})
-			framework.ExpectNoError(err, "failed to query for pod")
-
-			ginkgo.By("updating the pod to have a negative TerminationGracePeriodSeconds")
-			pod.Spec.TerminationGracePeriodSeconds = utilpointer.Int64(-1)
-			_, err = podClient.PodInterface.Update(ctx, pod, metav1.UpdateOptions{})
+			// about 26s timeout
+			updatePodBackoff := wait.Backoff{
+				Steps:    5,
+				Duration: 1 * time.Second,
+				Jitter:   2,
+			}
+			// retry if the TerminationGracePeriodSeconds is overrided
+			// see more in https://github.com/kubernetes/kubernetes/pull/115606
+			err = clientretry.RetryOnConflict(updatePodBackoff, func() error {
```

Comment:
just use retry.DefaultBackoff or retry.DefaultRetry (no strong opinion which)

Reply:
https://github.com/kubernetes/kubernetes/blob/93a51818714630429c603e0ec74873bb243daca8/staging/src/k8s.io/client-go/util/retry/util.go#L35-L43

I chosed `retry.DefaultBackoff` as the comments matched.